frontend/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -61,7 +78,7 @@ class BokehChart extends React.PureComponent<Props> {
     <div
       id={this.chartId}
       className="stBokehChart"
-      style={{ width: this.props.width }}
+      style={{ width: this.props.width, height: this.props.height }}
```

Comment:
What is this doing when `this.props.height` is undefined?

Reply:
solved